inst/WORDLIST

```diff
@@ -24,3 +24,4 @@ onwards
 pharmaverse
 prm
 th
+metadataset
```

Comment:
please move up to line 23 keep this file alphabetical for ease of maintenance

Reply:
Done